packstream/packstream-specification-1.md

```diff
@@ -182,7 +182,7 @@ Specifically, values between `00` and `7F` inclusive can be directly translated
 
 Similarly, values between `F0` and `FF` inclusive can do the same for negative numbers between -16 and -1.
 
-Note that while it is possible to encode small numbers in wider formats, it is generally recommended to use the most compact representation possible.
```

Comment:
The to-infinitive seemed right here to me.

Reply:
![Screenshot 2021-03-23 at 17 24 29](https://user-images.githubusercontent.com/445792/112181325-a15a4800-8bfc-11eb-859c-eadf34d3dcde.png)